### PR TITLE
Call model with additional args

### PR DESCRIPTION
**Pull request by dfalbel**
This PR enables passing additional arguments when calling a model. This enable something like:

```
model <- keras_model_sequential() %>% 
  layer_dropout(rate = 0.99999999)
model(1, training = TRUE)
model(1, training = FALSE)
```

### CODE REVIEW

**Comment by skeydan** [comment]
Cool!! Would we want to extend that possibility to Keras convenience functions (e.g., `predict`)? Or should we just require `call`ing a model?
(Just asking, don't even know if that would work in Python)

**Comment by dfalbel** [comment]
I think `predict` always assumes you are in non-training mode.

**Comment by skeydan** [comment]
Okay, so if you wanted to do e.g. prediction with dropout you'd just `call` instead, right?

**Comment by dfalbel** [comment]
Exactly!

Em sex, 1 de nov de 2019 às 10:47, Sigrid Keydana <notifications@github.com>
escreveu:
